src/components/TransactionDialogs/DelegationDialog/SelectValidators.tsx

```diff
@@ -0,0 +1,396 @@
+                            ) : null,
+                            endAdornment: (
+                              <InputAdornment position="end">
+                                <DropdownIcon />
+                              </InputAdornment>
+                            ),
+                          }}
+                        />
+                      )
+                    }}
+                  />
+                ) : (
+                  <div>Loading...</div>
+                )}
+              </Box>
+            ))}
+            <Box mt={1}>
+              <Button
+                variant="text"
+                color="secondary"
+                onClick={() =>
+                  setDelegations((d) => [
+                    ...d,
+                    { validator: '', amount: '', percentage: '', showSlider: false },
+                  ])
+                }
+              >
+                {t('add validator')}
+              </Button>
+            </Box>
+            <Box mt={2}>
+              <Typography gutterBottom>{t('memo')}</Typography>
+              <MemoInput
+                fullWidth
+                multiline
+                rows={3}
+                placeholder={t('description optional')}
+                InputProps={{
+                  disableUnderline: true,
+                }}
+                value={memo}
+                setValue={setMemo}
+                consent={consent}
+                setConsent={setConsent}
+              />
+            </Box>
+          </Grid>
+          <Grid item xs={6}>
+            <Typography gutterBottom>{t('amount')}</Typography>
+            {delegations.map((v, i) => (
+              <Box position="relative">
+                <Box key={i.toString()} display="flex" alignItems="center" mt={i === 0 ? 0 : 1}>
+                  <TextField
+                    sx={(t) => ({
+                      '.MuiInputBase-input': {
+                        padding: t.spacing(2),
+                      },
+                    })}
+                    fullWidth
+                    variant="filled"
+                    placeholder="0"
+                    type="number"
+                    InputProps={{
+                      disableUnderline: true,
+                      endAdornment: <InputAdornment position="end">{denom}</InputAdornment>,
+                    }}
+                    value={v.amount}
+                    onChange={(e) =>
+                      setDelegations((d) =>
+                        d.map((a, j) =>
+                          j === i
+                            ? {
+                                ...a,
+                                amount: e.target.value,
+                                percentage: ((100 * Number(e.target.value)) / amount).toFixed(2),
+                              }
+                            : a
+                        )
+                      )
+                    }
+                  />
+                  {/* isMobile condition: */}
```

Comment:
I'm not sure if this means everything under this is for mobile, but since the extension will only run in a desktop browser, there doesn't need to be any considerations for the mobile environement.